task10.py

```diff
@@ -0,0 +1,101 @@
+'''
+Print the following patterns. In the patterns ‘_’ (underscore) denotes the space. **_**
+'''
+#1
+n=int(input("Enetr a number :- "))
+
+'''
+1.
+    ** **
+    *   *
+        
+    *   *
+    ** **
+'''
+def diamond(n):
+    n= n
```

Comment:
Please remove this n=n, why you need this? This is not useful, as it not makes any sense here.